src/newrelic.js

```diff
@@ -1,89 +1,117 @@
+ */
+exports.config = {
+  /**
+   * Array of application names. Only applicable for local development because deployed
+   * environments use the env variable.
+   *
+   * @env NEW_RELIC_APP_NAME
+   */
+  app_name: ['tta-smart-hub-development'],
+  /**
+   * This setting controls distributed tracing.
+   * Distributed tracing lets you see the path that a request takes through your
+   * distributed system. Enabling distributed tracing changes the behavior of some
+   * New Relic features, so carefully consult the transition guide before you enable
+   * this feature: https://docs.newrelic.com/docs/transition-guide-distributed-tracing
+   * Default is true.
+   */
+  distributed_tracing: {
+    /**
+     * Enables/disables distributed tracing.
+     *
+     * @env NEW_RELIC_DISTRIBUTED_TRACING_ENABLED
+     */
+    enabled: true,
+  },
+  logging: {
+    /**
+     * Level at which to log. 'trace' is most useful to New Relic when diagnosing
+     * issues with the agent, 'info' and higher will impose the least overhead on
+     * production applications.
+     */
+    level: 'info',
+    filepath: 'stdout',
+  },
+  audit_log: {
+    enabled: process.env.NEW_RELIC_AUDIT_LOG_ENABLED
+      ? process.env.NEW_RELIC_AUDIT_LOG_ENABLED
+      : false,
+    endpoints: process.env.NEW_RELIC_AUDIT_LOG_ENDPOINTS
+      ? process.env.NEW_RELIC_AUDIT_LOG_ENDPOINTS
+      : [
+        'error_data',
+        'error_event_data',
+        'sql_trace_data',
+      ],
+  },
+  /**
+   * When true, all request headers except for those listed in attributes.exclude
+   * will be captured for all traces, unless otherwise specified in a destination's
+   * attributes include/exclude lists.
+   */
+  allow_all_headers: false,
+  attributes: {
+    /**
+     * Attributes to include from all destinations. Allows * as wildcard at end.
+     *
+     * NOTE: If including headers, they must be in camelCase form to be included.
+     *
+     * @env NEW_RELIC_ATTRIBUTES_INCLUDE
+     */
+    include: [
+      'request.headers.accept',
+      'request.headers.host',
+      'request.headers.referer',
+      'request.headers.userAgent',
+      'response.headers.contentType',
+      'response.headers.date',
+    ],
+    /**
+     * Prefix of attributes to exclude from all destinations. Allows * as wildcard
+     * at end. Will only have effect if `allow_all_headers` is true
+     *
+     * NOTE: If excluding headers, they must be in camelCase form to be filtered.
+     *
+     * @env NEW_RELIC_ATTRIBUTES_EXCLUDE
+     */
+    exclude: [
+      'request.headers.cookie',
+      'request.headers.authorization',
+      'request.headers.proxyAuthorization',
+      'request.headers.setCookie*',
+      'request.headers.x*',
+      'response.headers.cookie',
+      'response.headers.authorization',
+      'response.headers.proxyAuthorization',
+      'response.headers.setCookie*',
+      'response.headers.x*',
+    ],
+  },
+  error_collector: {
```

Comment:
I'd like to see comments for what we're adding that are as detailed as the existing ones, at the very least it would be nice to enumerate the options for the env variables as you very helpfully did in the PR description